src/js/utils/SchemaUtil.js

```diff
@@ -61,7 +61,7 @@ function schemaToFieldDefinition(fieldName, fieldProps, formParent, isRequired,
     definition.checked = value;
   }
 
-  if (fieldProps.fieldType === 'number') {
+  if (fieldProps.type === 'number') {
```

Comment:
I'm not sure what else needs to be changed, can you clarify? I believe @kennyt was saying that the schemas provided to the Install Modal already use `type` instead of `fieldType`.